shmem.c

```diff
@@ -199,7 +242,6 @@ bool init_shmem(void)
 	// Try to create shared memory object
 	shm_overTime = create_shm(SHARED_OVERTIME_NAME, size);
 	overTime = (overTimeDataStruct*)shm_overTime.ptr;
-	counters->overTime_MAX = (int) size;
 	initOverTime();
 
 	/****************************** shared settings struct ******************************/
```

Comment:
The settings struct is created in two places, this is the old place which needs to be removed (it was cherry-picked into the hotfix).